lib/main_common.pm

```diff
@@ -1086,7 +1086,7 @@ sub load_consoletests {
     }
     # salt in SLE is only available for SLE12 ASMM or SLES15 and variants of
     # SLES but not SLED
-    if (!is_staging && (is_opensuse || (check_var_array('SCC_ADDONS', 'asmm') || (sle_version_at_least('15') && !is_desktop)))) {
+    if (is_opensuse || (check_var_array('SCC_ADDONS', 'asmm') || (sle_version_at_least('15') && !is_desktop && !is_staging))) {
```

Comment:
Just to be safe, `check_var_array('SCC_ADDONS', 'asmm')` was previously combined with `!is_staging` check, so could you please add brackets, e.g.:
```perl
if (is_opensuse || !is_staging && (check_var_array('SCC_ADDONS', 'asmm') || sle_version_at_least('15') && !is_desktop)) {
```

Reply:
done